src/main/content/_assets/css/kabanero.scss

```diff
@@ -610,9 +611,30 @@ footer {
     position: relative;
 }
 
+#codewind_content{
```

Comment:
hyphens for class names

Reply:
that id isnt being used anymore so I just removed it